src/sentry/api/endpoints/organization_group_index_stats_unhandled.py

```diff
@@ -0,0 +1,129 @@
+from typing import List
```

Comment:
this is very similar to the existing stats endpoint, except it isn't collapsing "unhandled". this will be the endpoint we continue using once the frontend is updated. after that, the other one can be deleted.

Reply:
Which line is different? Also why make a whole new endpoint that only changes a little rather than changing the original one? What information is returned with `unhandled`? Sorry for all the questions I'm just coming in blind 😓

Reply:
so my updates are basically to add in the unhandled information to the response for the endpoint. I updated the serializer to do that. Because the front end currently uses the existing endpoint, that has been updated so that it doesn't add the unhandled information (because that endpoint doesn't need to add it). 

The new endpoint is basically the "old" version of the existing one so it doesn't collapse unhandled. 

I didn't want to change the existing one because doing so would unnecessarily take more time to find the unhandled info since this will be merged before any front end changes.

Reply:
Don't they both use the same serializer though? Maybe adding a test that shows the unhandled data that's returned would make this more clear.

Reply:
i was initially thinking about doing this without a feature flag, but after talking to Malachi, i think it would be easier to have one. so i'll just hide the unhandled bit behind the feature flag of the existing endpoint and update the tests!